packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.tsx

```diff
@@ -1,32 +1,39 @@
-/* tslint:disable */
```

Comment:
Should be TagItem.base.tsx

Reply:
@micahgodbolt I explained my choice in point 3 of my PR description. Do you still think we should have separate files for this tiny functional components? It's no issue to create them but I think this will add unnecessary bloatware when navigating through files of these leaf elements. Let me know if this is a must and I will make the change.

Reply:
Will leave it like this for now for easier reviews and maybe as the last PR will be some file organization.